core/dbt/events/types.py

```diff
@@ -655,6 +655,14 @@ def message(self) -> str:
 # Skipping E032, E033, E034
 
 
+class AdapterRegistered(InfoLevel):
+    def code(self):
+        return "E034"
+
+    def message(self) -> str:
+        return f"Registered adapter: {self.adapter_name}={self.adapter_version}"
```

Comment:
One last formatting issue with an extra `=`. not entirely sure where it's coming from but perhaps we need to get rid the extra one in this message?

```
17:10:01  Running with dbt=1.6.0-b4
17:10:01  target not specified in profile 'postgres', using 'default'
17:10:01  Registered adapter: postgres==1.6.0-b4
```

Reply:
hah didn't realize that our version specifier adds an `=` prefix